SparkPost/2web2ui
# FE-1140 - [Dashboard UX Redesign] Set up page plumbing

[FE-1140](https://jira.int.messagesystems.com/browse/FE-1140)

### What Changed
- Adds new V2 Dashboard page with a container for handling Redux
- Adds supporting tests

### How To Test
- Enable Hibana and the account UI option `allow_dashboard_v2`
- Visit `/dashboardV2` and verify the page renders
- Disable Hibana and visit `/dashboardV2` and verify the 404 page renders
- Re-enable Hibana and remove the `allow_dashboard_v2` - visit `/dashboardV2` and verify the 404 page renders

### To Do
